scripts/apps/workspace/directives/WorkspaceSidenavDirective.ts

```diff
@@ -41,11 +41,20 @@ export function WorkspaceSidenavDirective(superdeskFlags, Keys,
              * @param {object} e Gets $event from the element
              */
             scope.hideMonitoring = function(state, e) {
-                if (superdeskFlags.flags.authoring && state) {
-                    e.preventDefault();
-                    superdeskFlags.flags.hideMonitoring = !superdeskFlags.flags.hideMonitoring;
+                const fullWidthConfig: IFullWidthPageCapabilityConfiguration = scope.fullWidthConfig;
+
+                if (fullWidthConfig.enabled) {
+                    if (fullWidthConfig.allowed) {
+                        fullWidthConfig.onToggle(!scope.fullWidthEnabled);
+                    }
                 } else {
-                    superdeskFlags.flags.hideMonitoring = false;
+                    // eslint-disable-next-line no-lonely-if
+                    if (superdeskFlags.flags.authoring && state) {
+                        e.preventDefault();
+                        superdeskFlags.flags.hideMonitoring = !superdeskFlags.flags.hideMonitoring;
+                    } else {
+                        superdeskFlags.flags.hideMonitoring = false;
+                    }
```

Comment:
Asignment on line 54 always evaluates to `false`.

Reply:
How come? It would, if I was checking `superdeskFlags.flags.hideMonitoring` in the condition on line 52, but I'm checking for a different flag